### PR TITLE
CHANGELOG: Update changelog about modules instead of ./vendor

### PR DESCRIPTION
**Pull request by ptabor**
Follow up to PR: https://github.com/etcd-io/etcd/pull/12279

Please read https://github.com/etcd-io/etcd/blob/master/CONTRIBUTING.md#contribution-flow.

### CODE REVIEW

File: CHANGELOG-3.5.md [hunk 1]
```diff
@@ -228,6 +228,7 @@ Note that any `etcd_debugging_*` metrics are experimental and subject to change.
 
 - Require [*Go 1.15+*](https://github.com/etcd-io/etcd/pull/11110).
 - Compile with [*Go 1.15*](https://golang.org/doc/devel/release.html#go1.15)
+- Etcd uses go [modules](https://github.com/etcd-io/etcd/pull/12279) (instead of vendor dir) to track dependencies.
```
Comment (xiang90) [inline CHANGELOG-3.5.md:231]: Etcd -> etcd

**Comment by xiang90** [comment]
lgtm after fixing the nit.

File: CHANGELOG-3.5.md (hunk 1 above)
Comment (ptabor) [reply CHANGELOG-3.5.md:231]: Done

**Comment by xiang90** [comment]
@ptabor 

It is still `Etcd` not `etcd`. have you pushed your change?

**Comment by ptabor** [comment]
Fixed. I'm sorry. I misnamed the branch.

**Pull request merged**